### PR TITLE
W-15905026-kubernetes-crd-console-log-kt

### PR DESCRIPTION
**Pull request by kevintroller**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/customize-kubernetes-crd.adoc [hunk 1]
```diff
@@ -196,3 +197,16 @@ Key elements that can not be overwritten using the CRD:
...
+By default, console logging is disabled if it's detected that your Mule app logs at least one other source. You can override this default behavior using the Kubernetes template, either at the cluster level with the root Kubernetes template or at the Mule app namespace level.
+
+To enable console log using the Kubernetes template, set the `ENABLE_CONSOLE_LOG` variable in the `env` section:
+
+[source,copy]
+----
+pod:
+   env:
+       ENABLE_CONSOLE_LOG: "true"
+----
```
Comment (hegderohit89) [inline modules/ROOT/pages/customize-kubernetes-crd.adoc:216]: Can we also add this at line 120 ?
It should look something like this:

```
  pod:
    metadata:
      labels:
        podLabel: value
    env:
      ENABLE_CONSOLE_LOG: "true"
    affinity:
      nodeAffinity:
        preferredDuringSchedulingIgn

...
```

**Review by SparshAgarwal — DISMISSED** [review]

File: modules/ROOT/pages/customize-kubernetes-crd.adoc (hunk 1 above)
Comment (kevintroller) [reply modules/ROOT/pages/customize-kubernetes-crd.adoc:216]: Hey @hegderohit89 can you paste here the entire code?
Would it look like this?
```
pod:
    metadata:
      labels:
         podLabel: value
    env:
      ENABLE_CONSOLE_LOG: "true"
    affinity:
      nodeAffinity:
        preferredDuringSchedulingIgn
      annotations:
    automountServiceAccountToken: true
    containerSecurityContext:
      seccompProfile: 
        type: RuntimeDefault
    tolerations:
    - key: "example-key"
      operator: "Exists"
      effect: "NoSchedule"
    ephemeralStorageLimit: 4gb
    nodeSelector:
      region: us-east
    affinity:
    topologySpreadConstraints:
    - maxSkew: 1
      topologyKey: topology.kubernetes.io/zone
      whenUnsatisfiable: DoNotSchedule
      labelSelector:
        matchLabels:
          rtf.mulesoft.com/id: {{ .Values.id }}
```

Comment (hegderohit89) [reply modules/ROOT/pages/customize-kubernetes-crd.adoc:213]: yes that looks right

Comment (hegderohit89) [reply modules/ROOT/pages/customize-kubernetes-crd.adoc:216]: let me share entire code

Comment (hegderohit89) [reply modules/ROOT/pages/customize-kubernetes-crd.adoc:216]:
```
apiVersion: "rtf.mulesoft.com/v1"
kind: KubernetesTemplate
metadata:
  name: mule-application
  namespace: rtf
  labels:
spec:
  deployment:
    metadata:
      labels:
      annotations:
  service:
    metadata:
      labels:
      annotations:
  pod:
    metadata:
      labels:
      annotations:
    env: 
      ENABLE_CONSOLE_LOG: "true"
    automountServiceAccountToken: true
    containerSecurityContext:
      seccompProfile:
        type: RuntimeDefault
    tolerations:
    - key: "example-key"
      operator: "Exists"
      effect: "NoSchedule"
    ephemeralStorageLimit: 4gb
    nodeSelector:
      region: us-east
    affinity:
    topologySpreadConstraints:
    - maxSkew: 1
      topologyKey: topology.kubernetes.io/zone
      whenUnsatisfiable: DoNotSchedule
      labelSelector:
        matchLabels:
          rtf.mulesoft.com/id: {{ .Values.id }}
```

**Review by dmerlob — APPROVED** [review]

**Review by SparshAgarwal — APPROVED** [review]

**Review by Crispy-Salesforce — APPROVED** [review]

**Pull request merged**